### ISSUE TITLE
Seafile Client 6.1.8 on macOS 10.13.5: Context menu does not create Download Links

### ISSUE THREAD

**Issue by gn0rr** [haiwen/seafile-client#1102]
Clicking the menu item does nothing at all. Note that creating Internal Links does work. But to create a download link, the web-based UI has to be used.

**Comment by chytreg** [comment]
My and my team have the same problem as well

Hi, I've found that finder integration doesn't work for me.
When I click on `Get Seafile download link` from finder there is no action.

![image](https://user-images.githubusercontent.com/142751/42263864-826362fc-7f6f-11e8-81de-55936adcc265.png)

My Mac spec:

![image](https://user-images.githubusercontent.com/142751/42263889-9812a450-7f6f-11e8-88bd-c03aa8a78e03.png)

I the logs appreas only this message:
```
tail -f ~/.ccnet/logs/*.log

==> /Users/chytreg/.ccnet/logs/applet.log <==
[07/04/18 09:50:43]QObject::connect: No such signal GetSharedLinkRequest::success(const QString &)
```

I'm using the latest 6.1.8 client.
cc @bartekre

**Comment by freeplant** [comment]
It is fixed in 6.2.0-beta

**Comment by chytreg** [comment]
@freeplant Great thx, will try.
